src/sql/platform/theme/common/colorRegistry.ts

```diff
@@ -0,0 +1,83 @@
+export const buttonDropdownBorder = registerColor('buttonDropdownBorder', { light: 'transparent', dark: 'transparent', hc: '#2b56f2' }, nls.localize('buttonDropdownBorder', "The button dropdown border color"));
+export const buttonDropdownBackgroundHover = registerColor('buttonDropdownBackgroundHover', { light: '#efefef', dark: '#252525', hc: '#252525' }, nls.localize('buttonDropdownBackgroundHover', "The button dropdown border color"));
+export const extensionPackBorder = registerColor('extensionPackBorder', { light: '#8A8886', dark: '#8A8886', hc: '#2B56F2' }, nls.localize('extensionPackBorder', "The button dropdown border color"));
+export const extensionPackHeader = registerColor('extensionPackHeader', { light: '#fff', dark: '#fff', hc: '#fff' }, nls.localize('extensionPackHeader', "The button dropdown border color"));
+export const extensionPackBody = registerColor('extensionPackBody', { light: '#fff', dark: '#fff', hc: '#fff' }, nls.localize('extensionPackBody', "The button dropdown border color"));
+export const listBorder = registerColor('listBorder', { light: '#ccc', dark: '#ccc', hc: '#ccc' }, nls.localize('listBorder', "The button dropdown border color"));
+export const extensionPackHeaderShadow = registerColor('extensionPackHeaderShadow', { light: textShadow, dark: textShadow, hc: textShadow }, nls.localize('extensionPackHeaderShadow', "The button dropdown border color"));
+export const extensionPackGradientColorOneColor = registerColor('extensionPackGradientColorOne', { light: extensionPackGradientOne, dark: extensionPackGradientOne, hc: extensionPackGradientOne }, nls.localize('extensionPackGradientColorOne', "The button dropdown border color"));
+export const extensionPackGradientColorTwoColor = registerColor('extensionPackGradientColorTwo', { light: extensionPackGradientTwo, dark: extensionPackGradientTwo, hc: extensionPackGradientTwo }, nls.localize('extensionPackGradientColorTwo', "The button dropdown border color"));
+
+export const gradientOneColorOne = new Color(new RGBA(0, 0, 0, .2));
+
+export const gradientTwoColorOne = new Color(new RGBA(156, 48, 48, 0));
+export const gradientTwoColorTwo = new Color(new RGBA(255, 255, 255, 0.1));
+
+export const gradientOne = registerColor('gradientOne', { light: '#f0f0f0', dark: gradientOneColorOne, hc: gradientOneColorOne }, nls.localize('gradientOne', "The button dropdown border color"));
+export const gradientTwo = registerColor('gradientTwo', { light: gradientTwoColorOne, dark: gradientTwoColorTwo, hc: gradientTwoColorTwo }, nls.localize('gradientTwo', "The button dropdown border color"));
+export const gradientBackground = registerColor('gradientBackground', { light: '#fff', dark: 'transparent', hc: 'transparent' }, nls.localize('gradientBackground', "The button dropdown border color"));
+
+
+export const welcomePath = registerColor('welcomePath', { light: '#323130', dark: '#fff', hc: '#fff' }, nls.localize('welcomePath', "The border color for primary button"));
```

Comment:
please double check all the localized strings to make sure they are correct, i see a lot of duplicates

Reply:
fixed